chapters/en/chapter1/1.mdx

```diff
@@ -100,7 +100,42 @@ Of course! The course is released under the permissive [Apache 2 license](https:
 }
 ```
 
-## Let's Go
+## 🌎 Languages and translations[[languages-and-translations]]
+
+Thanks to our wonderful community, the course is available in many languages beyond English 🔥! Check out the table below to see which languages are available and who contributed to the translations:
+
+| Language                                                                      | Authors                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
It could be nice to use a Hugging Face account instead of GitHub for each contributor :)

Gently pinging the contributors in case they'd like to add their Hugging Face account (either in this PR or as a separate one) @lbourdois @ChainYo @melaniedrevet @abdouaziz @honghanhh @zhlhyx @petrichor1122 @1375626371 @avishek-018 @eNipu @JesperDramsch @MarcusFra @fabridamicelli @camartinezbu @munozariasjm @fordaz @jowharshamshiri @schoobani @pandyaved98 @omer-dor @gstdl @CaterinaBi @ClonedOne @Nolanogenn @EdAbati @hiromu166 @younesbelkada @HiromuHota @Doohae @wonhyeongseo @dlfrnaos19 @johnnv1 @victorescosta @LincolnVS @pdumin @svv73 @peeraponw @a-krirk @jomariya23156 @ckingkan @tanersekmen @mertbozkir @ftarlaci @akkasayaz @davidpeng86

Reply:
Good afternoon @lewtun, my huggingface account is [wseo](https://huggingface.co/wseo). Will contribute more in the days to come, thank you for the ping! Hope you have a great weekend 🤗

Reply:
Nice idea to include both accounts @wonhyeongseo - I'll add yours to this PR. Have a nice weekend too :)

Reply:
Hi, my account is [taners](https://huggingface.co/taners). I am happy to be part of this community. Thanks for your attitude 🤗

Reply:
Hi! My HF account is [resober](https://huggingface.co/resober)! Thank you! 💛

Reply:
Hey @lewtun, My HF Account is just the same - pandyaved98

Reply:
Hi, my HF account is tthhanh. Thanks.

Reply:
Hey, my hf acc is [johnnv](https://huggingface.co/johnnv)

Reply:
Hi, here is my HF account [ChainYo](https://huggingface.co/ChainYo)

Reply:
Sounds good, my HF account is [fordaz](https://huggingface.co/fordaz). Thanks @lewtun !

Reply:
Thank you @lewtun ! My HF account is [EdoAbati](https://huggingface.co/EdoAbati) :)

Reply:
Good Morning @lewtun . Here are both my Github and HF ids. Thanks for adding. Gonna contribute more.
Avishek: <a href="https://huggingface.co/avishek-018">GH</a> | <a href="https://huggingface.co/avishek-018">HF</a>

Reply:
thx @lewtun my HF account is [akkasayaz](https://huggingface.co/akkasayaz)

Reply:
Good idea! my HF account is [madatnlp](https://huggingface.co/madatnlp)

Reply:
Hey! Mu HF account is also [camartinezbu](https://huggingface.co/camartinezbu) 😄

Reply:
Thanks for the ping, my HF account is [gstdl](https://huggingface.co/gstdl)

Reply:
Hi there! My HF account is [ftarlaci](https://huggingface.co/ftarlaci). Thanks!

Reply:
My HF account is [yaoqi](https://huggingface.co/yaoqi). Thanks!